### ISSUE TITLE
serialport error when deploying flow

### ISSUE THREAD

**Issue by ThomasSiemens** [enocean-js/node-enocean#52]
I'm pretty new to that, so excuse me if there is already an answer in the other open issues, but so far, I can't solve the problem.

When I try to deploy a flow using the enocean listener I'll emediately get alot of errors in the debug screen, saying the serialport binding could not be  located. I guess that is a version/dependency thing?

What do I have to do to get it running? (described for a newbie)

Hoping for help and thanks in advance.
Thomas

```
Error: Could not locate the bindings file. Tried:
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/build/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/build/Debug/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/build/Release/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/out/Debug/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/Debug/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/out/Release/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/Release/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/build/default/serialport.node
 → /home/pi/.node-red/node_modules/node-enocean/node_modules/serialport/compiled/10.15.1/linux/arm/serialport.node
```

**Comment by Holger-Will** [comment]
i'm currently working on a complete rewrite of protocol implementation. https://github.com/enocean-js/enocean-js
the old node-enocean code is not maintained anymore. unfortunately node-red-contrib-enocean currently uses the old codebase. i'm not very much into node-red but it should be much easier now with the new codebase to implement a working node-red node. But  node-red-contrib-enocean will need a complete rewrite as well. i'll see if i find some time this weekend. 
It is not an error on your part and there is probably nothing you can do for now. i will let you know when i have something useable.

**Comment by ThomasSiemens** [comment]
Ok, that reassures me on one hand but make me stranded on the other! ;) So it would be awesome if you can find some time for a short fix for the meantime as your node would be the perfect solution for my project.

**Comment by ThomasSiemens** [comment]
@Holger-Will have you been able to find some time to figure out a workaround? Or no chance?

**Comment by BitTourette** [comment]
As je wrote, there won‘t be a workaround or a fix... he has to write/implement the new lib.

**Comment by ThomasSiemens** [comment]
Sorry, what I wanted to ask was, if there have been some time this weekend. ;)

**Comment by Holger-Will** [comment]
no, sorry no time yet... the lib is ready, but integrationg it into node-red has to be rewritten...

**Comment by Holger-Will** [comment]
https://github.com/enocean-js/enocean-js/tree/master/packages/node-red-contrib-enocean
